scripts/sync.py

```diff
@@ -126,6 +144,10 @@ def do(args):
             for Module in args.source:
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
+    # transfer new files after all sync attempts are done
+    if args.lochness_sync:
```

Comment:
It appears incomplete till now ... according to your change after my suggestion, it should be `args.lochness_sync_send`. I suppose you haven't tested after doing the modification? :p

Reply:
Thanks Tashrif- just updated this part. I'll put the testing module for the `sync.py` to test everything at the top abstraction level.

Reply:
I shall let you decide on the inclusion of this test, thank you.